bedrock/mozorg/helpers/misc.py

```diff
@@ -365,33 +361,33 @@ def donate_url(ctx, source=''):
 
     For en-US this would output:
 
-        https://sendto.mozilla.org/page/contribute/EOYFR2013-tabzilla
+        https://donate.mozilla.org/en-US/?presets=100,50,25,15&amount=50&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org&utm_medium=referral&utm_content=footer&currency=usd
 
     For de this would output:
 
-        https://sendto.mozilla.org/page/contribute/EOYFR2013-webDE
-
-    For fr this would output:
-
-        https://sendto.mozilla.org/page/contribute/EOYFR2013-webFR
+        https://donate.mozilla.org/de/?presets=100,50,25,15&amount=50&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org&utm_medium=referral&utm_content=footer&currency=eur
 
-    For pt-BR this would output:
+    For a locale not defined in settings.DONATE this would output:
 
-        https://sendto.mozilla.org/page/contribute/EOYFR2013-webPTBR
+        https://donate.mozilla.org/ca/?presets=100,50,25,15&amount=50&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org&utm_medium=referral&utm_content=footer&currency=usd
 
     """
     locale = getattr(ctx['request'], 'locale', 'en-US')
-    if locale not in settings.DONATE_LOCALE_LINK:
-        if 'default' in settings.DONATE_LOCALE_LINK:
-            locale = 'default'
-        else:
-            locale = 'en-US'
+    index = locale
+
+    # if the current locale is not available in settings.DONATE,
+    # we want to fallback to the default params but, still pass
+    # the current locale in the URL. This is a workaround whilst
+    # donate.m.o does not handle automaic locale negotiation.
+    # see https://bugzilla.mozilla.org/show_bug.cgi?id=1227199#c5
```

Comment:
```
# If the current locale is not available in settings.DONATE,
# fallback to the default params but still pass the current 
# locale in the URL. This is a temporary workaround since 
# donate.mozilla.org does not fully handle automatic
# locale negotiation.
```